### DISCUSSION TITLE
My Talks - siwei.io

### DISCUSSION THREAD

**Discussion by giscus** [siwei-io/siwei-io.github.io#20] [Comments]
# My Talks - siwei.io

DoK Talks #116 Nebula Graph: Open Source Distributed GraphDB Read more... Data on K8s Community 2021 GraphDB on Kubesphere Read more... K8s Community Day 2021 Openfunction + GraphDB Read more... COScon 2021 我的开源之路 Read more... PyCon China 2021 图数据库解谜与 Python 的图库应用实践 Read more... nMeetup: Nebula 应用上手实操 从头

https://siwei.io/en/talk/

**Comment by saber13812002** [comment]
I want to know more about you and your products. I want to learn more and use nebula graph in my projects. I want to develop open source plugins and addons for odoo and Laravel and others if I have supporter .I want to build website in IRAN for broadcast nebula for Iranians developer. I want to know more about developer advocate role in my team because I am DA in my team.

**Reply by wey-gu** [reply]
Dear Saber,

Sure, let's talk about it :), I saw you sent the request for the zoom meetup.
